Dockerfile

```diff
@@ -125,73 +125,15 @@ ENV DOCKER_TARGET=${DOCKER_TARGET}
 # Add our custom mime types (required for for ts/json/md files)
 COPY docker/etc/mime.types /etc/mime.types
 
-# Define production dependencies as a single layer
-# let's the rest of the stages inherit prod dependencies
-# and makes copying the /deps dir to the final layer easy.
-FROM base AS pip_production
-
-RUN \
-    --mount=type=bind,source=scripts/install_deps.py,target=${HOME}/scripts/install_deps.py \
-    # Files required to install pip dependencies
-    --mount=type=bind,source=./requirements/prod.txt,target=${HOME}/requirements/prod.txt \
-    # Files required to install npm dependencies
-    --mount=type=bind,source=package.json,target=/deps/package.json \
-    --mount=type=bind,source=package-lock.json,target=/deps/package-lock.json \
-    # Mounts for caching dependencies
-    --mount=type=cache,target=${PIP_CACHE_DIR},uid=${OLYMPIA_UID},gid=${OLYMPIA_UID} \
-    --mount=type=cache,target=${NPM_CACHE_DIR},uid=${OLYMPIA_UID},gid=${OLYMPIA_UID} \
-<<EOF
-${HOME}/scripts/install_deps.py prod
-EOF
-
 FROM base AS development
 
-FROM base AS locales
-ARG LOCALE_DIR=${HOME}/locale
-# Compile locales
-# Copy the locale files from the host so it is writable by the olympia user
-COPY --chown=olympia:olympia locale ${LOCALE_DIR}
-# Copy the executable individually to improve the cache validity
-RUN \
-    --mount=type=bind,source=requirements/locale.txt,target=${HOME}/requirements/locale.txt \
-    --mount=type=bind,source=Makefile-docker,target=${HOME}/Makefile-docker \
-    --mount=type=bind,source=locale/compile-mo.sh,target=${HOME}/compile-mo.sh \
-    make -f Makefile-docker compile_locales
-
-# More efficient caching by mounting the exact files we need
-# and copying only the static/ & locale/ directory.
-FROM pip_production AS assets
-
-# In order to create js i18n files with all of our strings, we need to include
-# the compiled locale files
-COPY --from=locales --chown=olympia:olympia ${HOME}/locale/ ${HOME}/locale/
-# TODO: only copy the files we need for compiling assets
-COPY --chown=olympia:olympia static/ ${HOME}/static/
-
-# Finalize the build
-# TODO: We should move update_assets to the `builder` stage once we can efficiently
-# Run that command without having to copy the whole source code
-# This will shave nearly 1 minute off the best case build time
-RUN \
-    --mount=type=bind,src=src,target=${HOME}/src \
-    --mount=type=bind,src=Makefile-docker,target=${HOME}/Makefile-docker \
-    --mount=type=bind,src=manage.py,target=${HOME}/manage.py \
-<<EOF
-echo "from olympia.lib.settings_base import *" > settings_local.py
-DJANGO_SETTINGS_MODULE="settings_local" make -f Makefile-docker update_assets
-EOF
+# Copy build info from info
```

Comment:
Because we now don't use the production image for development.. and due to the dependency graph here there is really no reason to maintain the split between locales/assets/production

They all just rely on one single command. That also bolsters the link between dev/prod because make up just re-runs that command to ensure host files match image files.

Reply:
<img width="1535" alt="image" src="https://github.com/user-attachments/assets/81192afb-6dc0-4588-81d3-9d64342d230e" />